apps/coscrad-frontend/src/components/resources/audio-item/audio-item-detail.thumbnail.presenter.tsx

```diff
@@ -3,34 +3,22 @@ import {
     ICategorizableDetailQueryResult,
     ResourceType,
 } from '@coscrad/api-interfaces';
-import { MediaPlayer } from '@coscrad/media-player';
-import { Link } from 'react-router-dom';
-import { routes } from '../../../app/routes/routes';
-import { FloatSpacerDiv, SinglePropertyPresenter } from '../../../utils/generic-components';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceDetailThumbnailPresenter } from '../../../utils/generic-components/presenters/detail-views';
 import { convertMillisecondsToSeconds } from '../utils/math';
-import './audio-item-detail.thumbnail.presenter.css';
 
 export const AudioItemDetailThumbnailPresenter = ({
     id,
+    name,
     lengthMilliseconds,
     audioURL,
-}: ICategorizableDetailQueryResult<IAudioItemViewModel>): JSX.Element => (
-    <div className="detail-thumbnail-container">
-        <Link to={`/${routes.resources.ofType(ResourceType.audioItem).detail(id)}`}>
-            <div
-                className="detail-thumbnail-meta-container"
-                title="View Connected Transcribed Audio"
-            >
-                <SinglePropertyPresenter display="Transcribed Audio ID" value={id} />
-                <SinglePropertyPresenter
-                    display="Duration"
-                    value={convertMillisecondsToSeconds(lengthMilliseconds)}
-                />
-            </div>
-        </Link>
-        <div className="detail-thumbnail-media-container">
-            <MediaPlayer audioUrl={audioURL} />
-        </div>
-        <FloatSpacerDiv />
-    </div>
-);
+}: ICategorizableDetailQueryResult<IAudioItemViewModel>): JSX.Element => {
+    return (
+        <ResourceDetailThumbnailPresenter id={id} name={name} type={ResourceType.audioItem}>
```

Comment:
adding the ResourceDetailThumbnailPresenter has meant removing the MediaPlayer for the thumbnail.  We can revisit how to add it back in.

Reply:
Ok. We want the Spotify experience in list views (e.g. connected resources panel) anyway.